### PR TITLE
feat: Support for renaming title of prompt.

### PR DESCRIPTION
**Pull request by Emt-lin**
Trying to address the problem that title of prompt can't change without altering the `CustomPrompt` structure.

I think the best solution is to change the `CustomPrompt` structure appropriately.
 
relation: #279

### CODE REVIEW

**Comment by logancyang** [comment]
Seems you are updating an old version. The code you are changing no long exists, please pull the latest code.

File: src/main.ts [hunk 1]
```diff
@@ -224,16 +224,35 @@ export default class CopilotPlugin extends Plugin {
               if (doc.prompt) {
                 new AddPromptModal(
                   this.app,
-                  (title: string, newPrompt: string) => {
-                    this.dbPrompts.put({
```
Comment (logancyang) [inline src/main.ts]: Seems you have an old version.

**Review by logancyang — CHANGES_REQUESTED** [review]
Seems you are updating an old version. The code you are changing no long exists, please pull the latest code.

**Comment by Emt-lin** [comment]
this PR is canceled.

**Pull request closed without merging**